modules/platforms/cpp/CMakeLists.txt

```
@@ -23,19 +23,74 @@ set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
 set(CMAKE_PROJECT_VERSION ${PROJECT_VERSION})
 
+option(ENABLE_CONAN "Install required libraries via Conan package manager" ON)
+option(ENABLE_CLIENT "Build Ignite.C++ Client module" ON)
+option(ENABLE_TESTS "Build Ignite.C++ tests" OFF)
+option(ENABLE_ADDRESS_SANITIZER "If address sanitizer is enabled" OFF)
+option(ENABLE_UB_SANITIZER "If undefined behavior sanitizer is enabled" OFF)
+option(WARNINGS_AS_ERRORS "Treat warning as errors" OFF)
+option(INSTALL_IGNITE_FILES "Install Ignite files" ON)
+
 set(CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake)
+list(APPEND CMAKE_MODULE_PATH ${CMAKE_BINARY_DIR})
+list(APPEND CMAKE_PREFIX_PATH ${CMAKE_BINARY_DIR})
+
+get_property(is_multi_config GLOBAL PROPERTY GENERATOR_IS_MULTI_CONFIG)
+
+set(conan_packages msgpack-c/4.0.0 gtest/1.12.1)
```

Let's keep it in a form "a library per line". This way it's easier to read and to track changes (i.e. with git).
```suggestion
set(conan_packages
    msgpack-c/4.0.0
    gtest/1.12.1
)
```